es6-lib/config/config.js

```diff
@@ -31,7 +31,7 @@ default {
   },
   heapDumpOut: '.',
 
-  emitProgressEvery: 200,
+  emitProgressEvery: 20,
```

Comment:
good call, considering your guesses were both incorrect :stuck_out_tongue: 

changed to `emitProgressEveryRows` - every N rows it does the thing
(and the thing is debounced - so we don't get more than 1 message every 2 seconds)